sideboard/websockets.py

```diff
@@ -24,6 +24,30 @@
 
 
 class threadlocal(object):
+    """
```

Comment:
MAGFest doesn't currently do anything with websockets so it doesn't care about anything in this file.  For the sake of completeness the three main changes in this file are:
- docstrings for all the utility methods that didn't have them
- support for passthrough subscriptions
- support for configurable/pluggable authentication